Automattic/woocommerce-payments
# Fix typo found in a heading on the new Deposits Overview component

#### Changes proposed in this Pull Request









There's a small typo that was found in the new Deposits Overview component. **Next deposits** should be singular **Next deposit**.

![image](https://user-images.githubusercontent.com/2275145/228709922-ee736902-ea1d-4e06-a07d-5a1298fb9a2c.png)

#### Testing instructions





1. In wp-options table, set `_wcpay_feature_simplify_deposits_ui` to 1
2. Navigate to Payments > Overview
3. Confirm the Deposits overview component has the correct "Next deposit" heading

-------------------


**Post merge**
